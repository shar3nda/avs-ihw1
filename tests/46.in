14
-1 1 0 9 1 9 -2 -1 6 10 7 4 0 -1