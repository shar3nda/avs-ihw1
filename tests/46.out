-2 2 5 10 2 2 1 7 -2 0